ConcernsCaseWork/ConcernsCaseWork.Shared.Tests/Factory/CaseFactory.cs

```diff
@@ -18,142 +18,148 @@ public static List<CaseDto> BuildListCaseDto(string trustUkPrn = null)
 			var dateTimeNow = DateTimeOffset.Now;
 			return new List<CaseDto>
 			{
-				// Status
-				// 1 - Live
-				// 2 - Monitoring
-				// 3 - Close
-				new CaseDto(
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					Fixture.Create<string>(),
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(),
-					trustUkPrn ?? Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					dateTimeNow, 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					1, 
-					1,
-					1
-				),
-				new CaseDto(
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					Fixture.Create<string>(),
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(),
-					trustUkPrn ?? Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					dateTimeNow, 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					2, 
-					3,
-					2
-				),
-				new CaseDto(
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					Fixture.Create<string>(),
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(),
-					trustUkPrn ?? Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					dateTimeNow, 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					3, 
-					2,
-					3
-				),
-				new CaseDto(
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					Fixture.Create<string>(),
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(),
-					trustUkPrn ?? Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					dateTimeNow, 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					4, 
-					1,
-					4
-				),
-				new CaseDto(
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					dateTimeNow, 
-					Fixture.Create<string>(),
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(),
-					trustUkPrn ?? Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					dateTimeNow, 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					Fixture.Create<string>(), 
-					5, 
-					2,
-					5
-				)
+				new CaseDto
```

Comment:
Pretty sure you can do a Fixture.Build to cut down on the number of lines needed, but not to worry

Reply:
Quite possibly - was going for a quick change with limited impact :- the main issue I had with this was to get rid of the massive constructor in the dto, and make it clear which values were being set